rebasehelper/output_tool.py

```diff
@@ -99,6 +96,19 @@ def print_cli_summary(cls, app):
             elif results_store.get_patches()['success']:
                 logger_output.error("\nPatching failed")
 
+    @classmethod
+    def check_for_important_checkers_output(cls):
```

Comment:
This doesn't seem right. It should be implemented in respective subclasses (checkers).

Reply:
Yes you are right. Then I am going to create a separate PR for it and rebase this branch onto it.